.github/workflows/validate-this-gha-refs.yml

```diff
@@ -3,7 +3,7 @@ on:
   push:
     branches:
       - dev
-  pull_request:
+  workflow_dispatch:
```

Comment:
Why is this and in `validate-this-pull-request` necessary?

Reply:
Just temporary!